### PR TITLE
Add section about installing conversations with custom session data

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As discussed in https://t.me/grammyjs/78172 and asked many times before.

### CODE REVIEW

**Review by roziscoding — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Review by habemuscode — APPROVED** [review]

File: site/docs/es/plugins/conversations.md [hunk 1]
```diff
@@ -326,6 +326,39 @@ bot.start();
 </CodeGroupItem>
 </CodeGroup>
 
+### Instalación con datos de sesión personalizados
+
+Ten en cuenta que si utilizas TypeScript y quieres almacenar tus propios datos de sesión además de utilizar conversaciones, tendrás que proporcionar más información de tipo al compilador.
+Digamos que tienes esta interfaz que describe tus datos de sesión personalizados:
+
+```ts
+interfaz SessionData {
```
Comment (CikiMomogi) [inline site/docs/es/plugins/conversations.md:335]: Typo `interface`?

**Review by CikiMomogi — DISMISSED** [review]

File: site/docs/es/plugins/conversations.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/es/plugins/conversations.md:335]: Yes, thanks

**Comment by KnorpelSenf** [comment]
@roj1512 isn't this ready?

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Pull request merged**